tests/trace/openai/test_instrumentor.py

```diff
@@ -0,0 +1,377 @@
+import json
+from importlib import reload
+
+import openai
+import pytest
+import responses
+from openai.error import AuthenticationError
+from phoenix.trace.openai.instrumentor import OpenAIInstrumentor
+from phoenix.trace.schemas import SpanException, SpanKind, SpanStatusCode
+from phoenix.trace.semantic_conventions import (
+    EXCEPTION_MESSAGE,
+    EXCEPTION_STACKTRACE,
+    EXCEPTION_TYPE,
+    INPUT_MIME_TYPE,
+    INPUT_VALUE,
+    LLM_FUNCTION_CALL,
+    LLM_INPUT_MESSAGES,
+    LLM_INVOCATION_PARAMETERS,
+    LLM_TOKEN_COUNT_COMPLETION,
+    LLM_TOKEN_COUNT_PROMPT,
+    LLM_TOKEN_COUNT_TOTAL,
+    MESSAGE_CONTENT,
+    MESSAGE_FUNCTION_CALL_ARGUMENTS_JSON,
+    MESSAGE_FUNCTION_CALL_NAME,
+    MESSAGE_NAME,
+    MESSAGE_ROLE,
+    OUTPUT_MIME_TYPE,
+    OUTPUT_VALUE,
+    MimeType,
+)
+from phoenix.trace.tracer import Tracer
+
+
```

Comment:
I think we should add a test to ensure that the instrumentation isn't invoked when not calling the `ChatCompletion` endpoint

Reply:
addressed in 36ff2b5f